notebooks/SITCOMTN_131_M1M3_VMS_TMA_Coupling_investigation.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>Describe this plot a little bit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-131/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>It was just a plot to check that the low pass filter was working and should have been removed --&gt;  removed now</p>